src/librustdoc/html/render/print_item.rs

```diff
@@ -1571,6 +1573,38 @@ fn document_type_layout(w: &mut Buffer, cx: &Context<'_>, ty_def_id: DefId) {
                     pl = if bytes == 1 { "" } else { "s" },
                 );
             }
+            if let Variants::Multiple { variants, .. } = &ty_layout.layout.variants {
+                if !variants.is_empty() {
```

Comment:
Can you test what happens for the empty case? Does it show 0 bytes or unsized? I think ideally it would show "uninhabited" or something.

Reply:
It shows 0 bytes. I think it would be more suitable for a separate PR as a followup of the original PR.